pallets/liquidity-pools/src/message.rs

```diff
@@ -229,7 +230,9 @@ pub enum Message<BatchContent = BatchMessages> {
 	/// Directionality: Centrifuge -> EVM Domain.
 	DisputeMessageRecovery {
 		/// The hash of the message which shall be disputed
-		hash: [u8; 32],
+		message: [u8; 32],
+		/// The address of the router
+		router: [u8; 32],
```

Comment:
Could you update the specs with this change? 🙏🏻 (or if there is a maintainer, the maintainer)

Reply:
This change was a result from syncing with the spec: https://centrifuge.hackmd.io/Soy4JiFbRSyovj53KW-a4A?view#DisputeMessageRecovery

Reply:
Ok! Somehow my hackmd was not updated with the latest version of that document